numba/tests/test_np_functions.py

```diff
@@ -124,14 +124,41 @@ def tril_m(m):
 def tril_m_k(m, k=0):
     return np.tril(m, k)
 
+def tril_indices_n(n):
+    return np.tril_indices(n)
+
+def tril_indices_n_k(n, k=0):
+    return np.tril_indices(n, k)
+
+def tril_indices_n_k_m(n, k=0, m=None):
+    return np.tril_indices(n, k, m)
+
+def tril_indices_from_arr(arr):
+    return np.tril_indices_from(arr)
+
+def tril_indices_from_arr_k(arr, k=0):
+    return np.tril_indices_from(arr, k)
 
 def triu_m(m):
     return np.triu(m)
 
-
 def triu_m_k(m, k=0):
     return np.triu(m, k)
 
+def triu_indices_n(n):
+    return np.triu_indices(n)
+
+def triu_indices_n_k(n, k=0):
+    return np.triu_indices(n, k)
+
+def triu_indices_n_k_m(n, k=0, m=None):
+    return np.triu_indices(n, k, m)
+
```

Comment:
Again, a two argument version (`n` and `m`) might be called for.

Reply:
Done